### PR TITLE
Replaced Socialize tab with footer icons

### PR DESCRIPTION
**Pull request by maurocon3ras**
## Description

Added Twitter and Facebook social icons at footer level so that you con Socialize from every page.
image

Added fontawesome cdn script to footer page.

Fixes # 275

### CODE REVIEW

**Comment by clairernovotny** [comment]
/cc @BethMassi @ddskier @sfaatz 

When working with the vendor earlier this year, the Social menu was a deliberate decision. Thoughts on this?

**Comment by ddskier** [comment]
I definitely like the idea of having social icons on every page. But I also like the idea of keeping the icons on the menu. Can we have both?

**Comment by isaacrlevin** [comment]
Can we move social icons to the footer AND have them in the Nav?

**Comment by maurocon3ras** [comment]
If you all agree I'd push changes to have them both 🤗

**Comment by maurocon3ras** [comment]
> /cc @BethMassi @ddskier @sfaatz
> 
> When working with the vendor earlier this year, the Social menu was a deliberate decision. Thoughts on this?

I just pushed changes to keep both the footer icons and nav tabs for socialize, what do you think?

File: input/_partials/_footer.cshtml [hunk 1]
```diff
@@ -40,3 +44,4 @@
     </div>
   </div>
 </footer>
+<script src="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"></script>
```
Comment (clairernovotny) [inline input/_partials/_footer.cshtml:47]: I thought we already had the font-awesome content included. Do we need this again here?

Comment (maurocon3ras) [reply input/_partials/_footer.cshtml]: Didn't realized at all... yeah, header has it, sorry! Just updated my PR with that correction

**Pull request merged**